node/hubController.js

```diff
@@ -290,49 +303,42 @@ class HubController {
             }
 
             if (!hubInfo) {
-                this.ConsoleLogger.error("invalid hub id");
+                logger.error("invalid hub id");
                 throw new OpenT2TError(404, OpenT2TConstants.InvalidHubId);
```

Comment:
If you're going to throw the error you don't need to also log (consuming implementations should be logging the Error themselves, or not depending on their preference, and the log version doesn't give any further info beyond the Error).  IFS logs the Error and can also generate telemetry on specific types of errors such as this.